components/sensor/teleinfo.rst

```diff
@@ -60,17 +60,17 @@ simply press -/+ buttons on the counter and look for `Standard mode` or
     sensor:
       - platform: teleinfo
         tags:
-         - name: "HCHC"
+         - tag_name: "HCHC"
```

Comment:
This `tag_name` issue has already been fixed in #871

Reply:
Oh! I didn't saw your PR because I was looking for something from @djtef ...
So yes, this include this little correction too.